stories/Form/Elements/Toggle.stories.tsx

```diff
@@ -63,7 +63,15 @@ const meta: Meta = {
 
 export default meta;
 
-const Template: Story<Props> = args => <ToggleComponent {...args} />;
+const Template: Story<Props> = args => {
```

Comment:
```suggestion
const Template: StoryFn<Props> = args => {
```

apparently, Story is deprecated